pycbc/workflow/matched_filter.py

```diff
@@ -243,12 +242,11 @@ def setup_matchedfltr_dax_generated_multi(workflow, science_segs, datafind_outs,
 
     if match_fltr_exe == 'pycbc_multi_inspiral':
         exe_class = select_matchedfilter_class(match_fltr_exe)
-        # Right ascension + declination provided in degrees,
-        # so convert to radians
+        # Right ascension + declination must be provided in radians
         cp.set('inspiral', 'ra',
-               str(radians(float(cp.get('workflow', 'ra')))))
+               cp.get('workflow', 'ra'))
         cp.set('inspiral', 'dec',
-               str(radians(float(cp.get('workflow', 'dec')))))
+               cp.get('workflow', 'dec'))
```

Comment:
Does this revert back to having to specify the RA/dec in the .ini file in radians rather than degrees?

Reply:
It does. This way it is directly consistent with `pycbc_multi_inspiral` that [asks](https://github.com/gwastro/pycbc/blob/master/bin/pycbc_multi_inspiral#L246) for RA and dec or the sky-grid in radians.

Undoing this change would not be a problem, but it would imply something ugly, i.e., we want RA and dec in degrees in the ini file, but the sky-grid file content to be in radians.

Reply:
My understanding of the decision was that all user-facing angles would be in degrees, but all internal operations would be in radians. As long as the GRB information is translated into a PyGRB `.ini` file by reviewable code, however, I guess it is not a big deal to just have everything in radians.

Reply:
Discussed briefly offline, we will go with radians for now.